website/docs/reference/widgets/code-scanner.md

```diff
@@ -6,15 +7,15 @@ The Code Scanner widget allows you to scan a variety of barcodes and QR codes. U
 
 ## Usage
 
-QR/Barcodes have taken off because they offer a clear and fast return on investment. It eliminates manual entry of product information at receiving, meaning there are far fewer opportunities for error. Let's look at some scenarios where a code scanner can be helpful:
+QR/Barcodes have taken off because they offer a clear and fast return on investment. It eliminates manual entry of product information at receiving, meaning there are far fewer opportunities for error. You can use a code scanner in some of the below scenarios:
```

Comment:
```suggestion
QR/Barcodes have taken off because they offer a clear and fast return on investment. It eliminates manual entry of product information at receiving, meaning there are far fewer opportunities for error. You can use a code scanner in the below scenarios:
```